package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@internetarchive/ia-menu-slider",
-  "version": "1.1.1",
+  "version": "1.1.3-a1",
```

Don't forget to revert this before cutting release! Note sure what the release flow is for this repo though.
```suggestion
  "version": "1.1.1",
```